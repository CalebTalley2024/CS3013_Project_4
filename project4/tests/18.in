5 1
10 2
4 2
6 1
7 4
20 2

