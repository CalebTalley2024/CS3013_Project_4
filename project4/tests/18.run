./scheduler PRIO tests/18.in 0
