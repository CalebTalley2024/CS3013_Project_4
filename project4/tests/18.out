Execution trace with PRIO:
Job 0 ran for: 5
Job 3 ran for: 6
Job 1 ran for: 10
Job 2 ran for: 4
Job 5 ran for: 20
Job 4 ran for: 7
End of execution with PRIO.
Begin analyzing PRIO:
Job 0 -- Response time: 0  Turnaround: 5  Wait: 0
Job 3 -- Response time: 5  Turnaround: 11  Wait: 5
Job 1 -- Response time: 11  Turnaround: 21  Wait: 11
Job 2 -- Response time: 21  Turnaround: 25  Wait: 21
Job 5 -- Response time: 25  Turnaround: 45  Wait: 25
Job 4 -- Response time: 45  Turnaround: 52  Wait: 45
Priority 1: Average -- Response: 2.50  Turnaround: 8.00  Wait: 2.50
Priority 2: Average -- Response: 19.00  Turnaround: 30.33  Wait: 19.00
Priority 4: Average -- Response: 45.00  Turnaround: 52.00  Wait: 45.00
Average -- Response: 17.83  Turnaround: 26.50  Wait: 17.83
End analyzing PRIO.
